spec.html

```diff
@@ -2700,6 +2700,28 @@ <h1>Well-Known Intrinsic Objects</h1>
                 ECMAScript Language Association
               </th>
             </tr>
+            <tr>
+              <td>
+                %AggregateError%
+              </td>
+              <td>
+                `AggregateError`
+              </td>
+              <td>
+                The `AggregateError` constructor (<emu-xref href="#sec-aggregate-error-constructor"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %AggregateErrorPrototype%
+              </td>
+              <td>
+                `AggregateError.prototype`
+              </td>
+              <td>
+                The initial value of the `prototype` data property of %AggregateError%; i.e., %AggregateError.prototype%
+              </td>
+            </tr>
```

Comment:
Note that <https://github.com/tc39/ecma262/pull/2056> will remove the legacy non‑dotted aliases.

Reply:
Whichever lands second can rebase and adapt to the one that lands first.